packages/server/scripts/exportAppTemplate.js

```diff
@@ -22,18 +25,22 @@ yargs
       },
     },
     async args => {
+      const name = args.name,
+        appId = args.appId
       console.log("Exporting app..")
-      if (args.name == null || args.appId == null) {
+      if (name == null || appId == null) {
         console.error(
           "Unable to export without a name and app ID being specified, check help for more info."
         )
         return
       }
-      const exportPath = await exportTemplateFromApp({
-        templateName: args.name,
-        appId: args.appId,
-      })
-      console.log(`Template ${args.name} exported to ${exportPath}`)
+      const exportPath = join(budibaseAppsDir(), "templates", "app", name, "db")
+      fs.ensureDirSync(exportPath)
+      const writeStream = fs.createWriteStream(join(exportPath, "dump.text"))
```

Comment:
Is the budibase home directory always the one used for these temporary file ops? Cause I know there's a budibase temp dir too. Just wondering when running the server in dev now (outside docker), what are the actual file paths that are used. Are the home dir and temp dir both still used?

Reply:
That old script doesn't really have a purpose anymore, to be honest its not entirely possible to script it that way, realistically need to rewrite that script - although you can export from an endpoint on the server. I might actually just remove the script cause its just not useful anymore.

On your question, those directories outside docker can never be hit, we don't actually use the apps directory even in development with the server running externally, everything that happens happens in the OS system level temporary files directory (unix `/tmp`). The home directory is still technically used to download automations just so they live on somewhere between reboots etc and aren't wiped out in anyway shape or form but apart from that all files which get created should exist in the `/tmp/.budibase` directory.